server/src/termdb.server.init.js

```diff
@@ -61,6 +61,8 @@ export function server_init_db_queries(ds) {
 		'anno_categorical',
 		'buildDate'
 	]
+	if (tables.has('sample_ancestry') && cn.prepare('SELECT * FROM sample_ancestry').all().length > 0)
```

Comment:
@airenzp Should eventually improve this to `SELECT count(*) as num_rows FROM sample_ancestry`, which would return only one row versus returning potentially thousands of rows.